sentry/src/main/java/io/sentry/hints/AttachmentContainer.java

```diff
@@ -0,0 +1,39 @@
+package io.sentry.hints;
+
+import io.sentry.Attachment;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class AttachmentContainer {
```

Comment:
Split this out from `Hints` to not bloat it and also make it easier for us to maintain.

Reply:
I think this could be an inner class of `Hint`. It can only exist inside of `Hint`. As this is public, I think the focus should rather be on the API and not maintenance. Then we could also name it `Attachments`, which would turn into `Hint.Attachments`.

Reply:
I just haven't come up with a good name yet, so I chose `AttachmentContainer`, I like `Attachments`. Can change the name, not sure about making this an inner class, what do the others think?.